modules/euidIdSystem.js

```diff
@@ -99,6 +99,14 @@ export const euidIdSubmodule = {
       internalStorage: ADVERTISING_COOKIE
     };
 
+    if (FEATURES.UID2_CSTG) {
```

Comment:
I think re-using this feature flag is fine - I expect any publisher using CSTG for either module would almost certainly use it for both modules.